build/bootstrap/bootstrap-debian-10.sh

```diff
@@ -5,13 +5,17 @@ set -e
 
 echo "Setting up development environment for do-client"
 
-# Various development machine tools 
+# Various development machine tools
 apt-get update -y --fix-missing
 apt-get install -y make build-essential g++ gdb gdbserver gcc git wget
 apt-get install -y python3 cmake ninja-build
 
 # Open-source library dependencies
-apt-get install -y libboost-all-dev libgtest-dev libproxy-dev libmsgsl-dev libssl-dev uuid-dev
+# Boost libs for DO
+apt-get install -y libboost-system-dev libboost-log-dev libboost-filesystem-dev libboost-program-options-dev
+# Additional Boost libs for cpprestsdk
+apt-get install -y libboost-random-dev libboost-regex-dev
```

Comment:
We're still pending the task to remove cpprest for the http server implementation within the agent correct?

Reply:
Yes, I'm leaving that for Nickel.